### PR TITLE
Feature - The Todo List 

### PR DESCRIPTION
**Pull request by rubenpazch**
1. Your ‘todos’ are going to be objects that you’ll want to dynamically create, which means either using factories or constructors/classes to generate them.

2. Brainstorm what kind of properties your todo-items are going to have. At a minimum they should have a title, description, dueDate and priority. You might also want to include notes or even a checklist.

3. Your todo list should have projects or separate lists of todos. When a user first opens the app, there should be some sort of ‘default’ project to which all of their todos are put. Users should be able to create new projects and choose which project their todos go into.

4. You should separate your application logic (i.e. creating new todos, setting todos as complete, changing todo priority etc.) from the DOM-related stuff, so keep all of those things in separate modules.

5. The look of the User Interface is up to you, but it should be able to do the following

- view all projects

- view all todos in each project (probably just the title and duedate.. perhaps changing color for different priorities)

- expand a single todo to see/edit its details

- delete a todo

6. Use localStorage to save user’s projects and todos between sessions.

### CODE REVIEW

File: src/project/projectModule.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { projectFactory } from './projectFactory';
+// import { CreateDomElement } from '../helpers/createDomElements';
+// import { ConfigurePage } from '../helpers/configurePage';
```
Comment (maelfosso) [inline src/project/projectModule.js]: Kindly remove comments. They are not allowed.

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+```
+npm install --global webpack
+```
+
+You can then compile the code with the next command
+
+```
+npm start
+```
+After this, a main.js file will be created and written in the dist folder. Copy and paste index.html and main.js wherever you want and you can now see them in your browser.
```
Comment (maelfosso) [inline README.md]: `Copy and paste ... your browser` I don't think it's the good way to start the application.
Find a simple way to do it.

Also I suggest you install and set up `webpack-dev-server` which easy the launch of the application.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @MiguelDP4 @rubenpazch 

You have an original approach !! Really!! It's the first time that I see such a UI/UX for this project
However, there are some weird interactions that don't seem good

1. I manage to understand how it works. I mean if i clicked on **complete** button the TODO will disappear because it's complete of course.
But something I don't understand it's **why do the complete todos appear when I refresh the browser?** Don't they supposed to stay hidden? Or you didn't keep their status as complete. I think, from what I'm seeing from the localStorage that it's because you don't have a task's status field

2. Please, how do the user do to delete a TODO? It's a requested feature. Check it [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png)

3. Read the comments left below.

4. There is two HTML files `todo-list-v2.html` and `todo-list.html`. I don't think you still use them. Please, delete them.


Kindly make the changes and submit a new code review request
Thanks

Mael FOSSO

File: src/project/projectModule.js (hunk 1 above)
Comment (rubenpazch) [reply src/project/projectModule.js]: thank this was changed

File: README.md (hunk 2 above)
Comment (rubenpazch) [reply README.md]: solved

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap:

Thank you for applying the changes
You can merge it now :+1: 

Good continuation :handshake: 

Mael FOSSO

**Pull request merged**